changelog/next/bug-fixes/4012--multiline-syslog.md

```diff
@@ -0,0 +1 @@
+Parsing a multi-line syslog message no longer causes a crash.
```

Comment:
This crash is not caused by multi-line messages, but whenever `unknown_syslog_builder` is used (including single-line messages).